tests/rules/test_omnienv_no_such_command.py

```diff
@@ -26,7 +26,7 @@ def Popen(mocker):
 
 @pytest.mark.parametrize('script, pyenv_cmd', [
     ('pyenv globe', 'globe'),
-    ('pyenv intall 3.8.0', 'intall'),
+    ('pyenv install 3.8.0', 'install'),
```

Comment:
To better understand the usage of `pyenv_cmd`, check the `output` fixture up top.